### ISSUE TITLE
키워드 전처리 및 분류

### ISSUE THREAD

**Issue by chuminji2** [chuminji2/youtube-trend-analyzer#2]
from sklearn.feature_extraction.text import TfidfVectorizer
from sklearn.cluster import KMeans

# 텍스트 전처리 후 클러스터링
vectorizer = TfidfVectorizer()
X = vectorizer.fit_transform(keyword_list)
model = KMeans(n_clusters=5)
labels = model.fit_predict(X)
